README.md

```diff
@@ -4,15 +4,42 @@ Are you ready to ask your [Amazon Echo](http://www.amazon.com/echo), *"Alexa, wh
 
 This project is an implementation of the [OneBusAway](http://onebusaway.org/) open-source platform for real-time transit info for Amazon's [Alexa Voice Service](https://developer.amazon.com/public/solutions/alexa/alexa-voice-service).
 
-### Install and Use
+## Install and Use ##
 
 This project isn't released publicly yet.  We plan to make the first release in early 2016.
 
-Initial spoken setup: TBD
+To both configure and use, just start with `open onebusaway`.
 
-Supported voice commands: TBD
+Refer to `interaction model/utterances.txt` for the full list of spoken phrases we support.
 
-### Build
+## Develop ##
+
+The application backing the skill was designed to run in AWS.
+
+### Prepare your AWS environment
+
+1. Log in to your [AWS Console](http://console.aws.amazon.com) and switch to the "N. Virginia" region.
+1. Apply the CloudFormation template in `aws/cloudformation/onebusaway.template` in your AWS account.
+   Name your stack "onebusaway-alexa" for consistency with existing documentation.
+1. Acknowledge "that this template might cause AWS CloudFormation to create IAM resources."
+1. Switch to the Output tab of your stack and note the AWS key ID and secret key.
+   You will use these in the next section.
+
+### Personalize the app
+
+1. Create `src/main/resources/onebusaway.properties` with the following content:
+
+```
+skill-app-id=amzn1.echo-sdk-ams.app...

```

Comment:
I'd suggest adding a note here referencing the below section (assuming my other comment is correct) - something like:
`skill-app-id=amzn1.echo-sdk-ams.app...   // You'll get this when setting up your Skill.  See the below section "Deploy to your Alexa device for the first time".`